app/controllers/manual_sections_controller.rb

```diff
@@ -1,3 +1,9 @@
 class ManualSectionsController < ApplicationController
+  def show
+    @section = Section.find(content_id: params[:content_id])
```

Comment:
I've followed a similar approach for `Manual.where` so it would make sense to do this in `Section.find`. I haven't got around to adding 404s for all these AR like things, but it's on a post it note on the board.